templates/layouts/mooneye-install.hbs

```diff
@@ -66,34 +66,55 @@
         Navigate to that folder using the Windows commandline (cmd.exe).
         <br><br>
       </div>
-
       <h1>Instructions</h1>
-      <h3>1. Unlock your bootloader</h3>
-      Installing AsteroidOS requires an unlocked bootloader. Accessing the bootloader can be achieved via ADB in Wear OS, as described in steps 1.1 to 1.4 below.
-      <br>You can skip these steps if you are familiar with the manual method to access the bootloader. This might need some practice and timing. Hold down the power until the watch reboots. When the MOBVOI logo appears, quickly slide on the screen from the upper left corner to the lower right corner.
-      <br>
-      <br>Enable ADB on your watch with the following steps:
-        <ol>
-          <img src="{{assets}}/img/install-1.jpg" class="install-centered-img"/>
-          <li>Open the settings app and go to the <b>System</b> and then <b>About</b> page.</li>
-          <li>Tap the <b>Build number</b> 7 times until Developer options become available.</li>
-
-          <img src="{{assets}}/img/install-2.jpg" class="install-centered-img"/>
-          <li>Go back to the settings menu and then to the <b>Developer options</b> page.</li>
-          <li>Enable <b>ADB debugging</b>.</li>
-        </ol>
-      <br><b>Finally, to unlock the bootloader</b> enter the following commands in a terminal with your watch connected:</li>
+      <h2>1. Unlock your bootloader</h2>
+      Installing AsteroidOS requires an unlocked bootloader.
+      <br>Accessing the fastboot bootloader menu on your watch can be achieved via ADB in Wear OS, as described below in step 1.1.
+      <br>You can skip to step 1.2 if you are familiar with the manual method to access the bootloader menu. The methods are explained on the <a href="https://asteroidos.org/wiki/useful-commands/#boot-to-fastboot-bootloader-menu">useful commands page</a>.
+      <h3>1.1 Enable ADB on your watch with the following steps:</h3>
+      <div class="unlock-adb-wrapper">
+        <div class="unlock-step-wrapper">
+          <img src="{{assets}}/img/install-aw-01.webp" class="unlock-step-img">
```

Comment:
For some reason this one image loads as a grey square for me, do you have the same issue ? https://raw.githubusercontent.com/AsteroidOS/asteroidos.org/e00728d931cd2c3c28fda35df89931141ff49897/assets/public/img/install-aw-01.webp

Other images work

Reply:
Yeah, thanks for confirming. I had a similar occurrence, but in my case the 1st and 5th image would not show on firefox at some point. I did ignore the issue since just opening the same state in any other brower then showed all images correctly.
We might just move back to png since this smells like a webp problem?...

Reply:
Ok, i guess i found the issue. That first picture was the only one i exported to "lossless" webp which seems to be not perfectly supported by all browsers. Now exported it to 90% image quality like the other and the problem disappeared for me.

Reply:
Neat, indeed the new picture works for me too.